src/main/java/my/project/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,35 @@
+package my.project.lotto.domain;
+
+import my.project.constants.Rule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created : 2020-11-23 오전 11:24
+ * Developer : Seo
+ */
+public class Lotto {
+    private List<GameRecord> records;
+
+    public Lotto() {
+        this.records = new ArrayList<>();
+    }
+
+    public List<GameRecord> games(GameInfo info) {
+        for (int i = 0; i < info.getCount(); i++) {
+            game();
+        }
+        return records;
+    }
+
+    private void game() {
+        Machine machine = new Machine();
+        GameRecord record = new GameRecord();
+        for (int i = 0; i < Rule.ONE_GAME_COUNT; i++) {
+            record.add(machine.pop());
+        }
+        records.add(record);
```

Comment:
```suggestion
        GameRecord record = Machine.generateLotto();
        records.add(record);
```
이렇게 수행되도록 만들어보면 어떨까요?

혹은

```java
        records.add(Machine.generateLotto());
```
이렇게도 가능할 것 같습니다.!